### ISSUE TITLE
Remote headers

### ISSUE THREAD

**Issue by bhack** [Azure/vscode-kubernetes-tools#403]
How intellisense could resolve remote headers of librariea installed in the container image build and so part of  the running pod?

**Comment by itowlson** [comment]
I'm not quite sure what this is requesting.  Are you thinking about debugging into shared libraries that are on the container but for which the client does not have debugging symbols?

**Comment by bhack** [comment]
I meant something like having a pod image where app dependecies headers are installed with i.e apt-get <librarypackage-dev> in the docker build phase. How intellisense can parse that headers that are image dependent and available only inside the pod filesystem. See also https://github.com/atom/atom/issues/14860#issuecomment-437638762

**Comment by bhack** [comment]
I've partially open another related ticket at https://github.com/Microsoft/vscode-cpptools/issues/2818. 
But I think that the problem is more general and related to other libraries of other languages in the image (Dockerfile with apt, pip, conda etc.. RUN steps)

**Comment by itowlson** [comment]
Yes, I think this is a broader issue, which I think is what initially confused me.  The server part of the debugger needs to provide this stuff, and the client part needs to be able to work with the provided symbols, but that's going to be very different from language to language and even implementation to implementation, and requires involvement from the debugger component not just the k8s component.

**Comment by bhack** [comment]
Yes probably we need to just verify here that the ports of the deployed object are opened in the dev/debug phase.

**Comment by bhack** [comment]
See also https://github.com/Microsoft/vscode/issues/29194

**Comment by bhack** [comment]
Ok the final placeholder for this Is https://github.com/microsoft/vscode-remote-release/issues/12
